docs/types/txikijs.d.ts

```diff
@@ -599,24 +585,16 @@ declare global {
         * @param path The path to of the directory to be created.
         * @param options Options for making the directory.
         */
-        function mkdir(path: string, options?: MkdirOptions): Promise<void>;
+        function makeDir(path: string, options?: MkdirOptions): Promise<void>;
         
         /**
         * Copies the source file into the target.
         *
-        * If `COPYFILE_EXCL` is specified the operation will fail if the target exists.
```

Comment:
I am not supportive of removing these flags as they have utility. I would rather add configuration flags in `CopyFileOptions` to reflect them.

Reply:
I have never needed them and they are so low level I'm not sure they have a use in the context where txiki.js is used. Adding an options object afterwards will be no problem.